packages/edit-site/src/components/page-templates/dataviews-templates.js

```diff
@@ -94,6 +110,32 @@ function AuthorField( { item } ) {
 	);
 }
 
+function TemplatePreview( { content, viewType } ) {
+	const settings = usePatternSettings();
```

Comment:
It's a bit weird that we use this hook when this page is unrelated. Looking at the data the hook provides, it sounds like we'd only need the settings from the site editor (everything else is related to patterns). Can we pull those settings directly here?

Reply:
This hook is misnamed I think. It's about providing the block editor with the settings it needs to render anything (with all the patterns, styles...)

Reply:
Actually this hook loads the patterns are needed here too, in order to render previews with patterns inside. These patterns are loaded within a Block Editor instance and in our case we don't load one.